README.md

```diff
@@ -21,6 +21,7 @@ Usage:
 ```
 --US            site list for US region
 --EU            site list for European region
+--chrome        select Chrome browser, Firefox as default
```

Comment:
lets add `--firefox` and check if only one firefox or chrome is selected.

Reply:
Added.